apps/web/src/api/auth.ts

```diff
@@ -1,5 +1,6 @@
 export const loginUser = async (email: string, password: string) => {
-  const response = await fetch("/api/v1/auth/login", {
+  const base = import.meta.env.VITE_LOGIN_API_URL;
+  const response = await fetch(base, {
```

Comment:
This should be:

`const base = import.meta.env.VITE_API_URL; 

fetch(${base}/auth/login, ...) `

your .env should have `VITE_API_URL=/api/v1/`

Reply:
probably should have the base url as well, unless it works already